### ISSUE TITLE
quantuminspire.qiskit.set_project_name not working?

### ISSUE THREAD

**Issue by quantumAnsatz** [QuTech-Delft/quantuminspire#95]
# Bug report

## What did you try to do?

I tried to assign a project name using quantuminspire.qiskit just after the authentication but I keep receiving an error message.

## What did you expect to happen?

I wanted to assign a project name as explained in the README.md

## What actually happened?

Here is the code
```
from quantuminspire.qiskit import QI
QI.set_authentication()
QI.set_project_name('my-project-name')
```
authentication is done through an API token and works well, but then I get the following error
```
QI.set_project_name('my-project-name')
AttributeError: 'QuantumInspireProvider' object has no attribute 'set_project_name'
```

## Environment

Please specify:
* operating system, version and distribution: MacOs Catalina
* python version (`python --version`) and distribution: python3.9
* list of installed packages (`pip list` or equivalent): qiskit, quantuminspire

**Comment by QFer** [comment]
Hi quantumAnzats,

Setting the project name is functionality not yet released (to pypi) and only available in our dev(elopment) branch.
If you want to use this functionality please install quantum inspire from source.

```
git clone https://github.com/QuTech-Delft/quantuminspire
cd quantuminspire
pip install .[qiskit]
```

Best regards,
 
Fer Grooteman
Quantum Inspire Support Team

**Comment by QFer** [comment]
@quantumAnsatz, Good news. Today we released version 1.4.0 of our SDK where the project name functionality as you used it is now part of. We updated pypi.

**Comment by quantumAnsatz** [comment]
I just tried the new version and seems to work correctly. Thank you very much for your attention.
